ClientKit/ClientKit.cs

```diff
@@ -352,6 +352,17 @@ protected virtual void Dispose(bool disposing)
                 }
             }
 
+            /// <summary>
+            /// Get a parsed display configuration. This lets you query eyes, surfaces, and
+            /// viewers.
+            /// </summary>
+            public DisplayConfig GetDisplayConfig()
+            {
+                SafeDisplayConfigHandle handle;
+                DisplayConfigNative.osvrClientGetDisplay(this.m_context, out handle);

```

Comment:
Note that this (native call) doesn't always succeed - if we haven't got a display config yet because the path tree hasn't shown up, or if the display config is corrupted, this will return a failure and not touch the output parameter.